### DISCUSSION TITLE
I am working on pyhanko , this working fine on my local machine but getting IsADirectoryError on production.

### DISCUSSION THREAD

**Discussion by ratnamkamlesh** [MatthiasValvekens/pyHanko#232] [Q&A] [answered]
Hi Matthias,
 I am working on pyhanko, this working fine on my local machine but getting below error on production:-
IsADirectoryError: [Errno 21] Is a directory: '/etc/localtime'

**Comment by MatthiasValvekens** [comment, marked as answer]
(Moved from issue tracker since this is not an actionable bug report. Please read the [contribution guidelines](https://github.com/MatthiasValvekens/pyHanko/blob/master/CONTRIBUTING.md#use-of-the-issue-tracker-and-discussion-forum) before posting.)

Well, what can I say, your local timezone file is a directory for some reason. That's an issue with your environment, and has virtually nothing to do with pyHanko. For what it's worth, the dependency that tries to access that file is probably `tzlocal` (which is heavily platform-dependent).
